### DISCUSSION TITLE
Comments sections/lines in a .env file

### DISCUSSION THREAD

**Discussion by miyacoz** [goatandsheep/react-native-dotenv#149] [Q&A] [answered]
imagine we have a .env file like below:

```sh
SOME_KEY=123
SOME_API_KEY=aabbcc # some note
```

then, the app reads the `SOME_API_KEY` as a string value `aabbcc # some note`.
is it a planned behaviour? it doesn't seem it's documented at least on the README.md.

i want it to be simply `aabbcc` after removing `#` and the part after it, and trimming spaces, but i'm fine even if it's working as you planned, i can sure do workarounds :)

**Comment by goatandsheep** [comment, marked as answer]
You cannot mix values and #comments on the same line. This is part of dotenv unfortunately. However you can on separate lines

**Reply by miyacoz** [reply]
thank you for answering! understood :)
